packages/terra-graphs-docs/src/example-datasets/dataObjects/Gantt/simplePanningData.js

```diff
@@ -0,0 +1,49 @@
+import { loadTaskPopup, loadXAndYAxisLabelPopup } from '@cerner/terra-graphs-docs/lib/example-datasets/popup';
+import { loadPopup } from '@cerner/carbon-graphs/dev/examples/popup';
+
+const tasks2 = [
```

Comment:
Why is this named `task2`?

Reply:
Changed name [here](https://github.com/cerner/terra-graphs/pull/176/commits/2c4e54552109d0cea59d4ab0fd1897a6c78f2762#diff-5f824e9f059845c37bed5c8db182769934e4d86e6bef513b5097ebf84afcdc60R4).